apps/ios/LiveVideo/LiveVideo/Launch/LiveVideoApp.swift

```diff
@@ -29,15 +29,15 @@ struct LiveVideoApp: App {
                     let localParticipant = LocalParticipantManager(authManager: authManager)
                     let roomManager = RoomManager()
                     roomManager.configure(localParticipant: localParticipant)
-                    let viewerStore = ViewerStore()
-                    let speakersStore = SyncUsersStore()
-                    let raisedHandsStore = SyncUsersStore()
-                    let viewersStore = SyncUsersStore()
+                    let userDocument = SyncUserDocument()
+                    let speakersMap = SyncUsersMap()
+                    let raisedHandsMap = SyncUsersMap()
+                    let viewersMap = SyncUsersMap()
```

Comment:
I had to rename some other things in iOS because I was dealing with `SyncUsersStore` history which is a reusable way to load sync maps that contain users.

During the rename I made things a little less abstract which I maybe wouldn't do in a real production app but I think it will be easier for customers to follow in this app.